src/ploomber/tasks/notebook.py

```diff
@@ -913,6 +934,10 @@ def run(self):
             ]
         )
 
+        cwd = str(self.source.loc.parent.resolve())
+        orig_env = os.environ.copy()
+        orig_env["PYTHONPATH"] = cwd
```

Comment:
this will overwrite any existing value, we should append the new value if PYTHONPATH exists